v8.go

```diff
@@ -184,9 +200,11 @@ type Context struct {
 	ptr C.ContextPtr
 
 	callbacks      map[int]callbackInfo
+	callbacksMutex sync.RWMutex
```

Comment:
A Context isn't safe for concurrent usage, so I don't think we need these mutexes -- you shouldn't be accessing them from more than one goroutine at all.